lib/lambda/sinkMainProcessors.ts

```diff
@@ -197,6 +204,13 @@ export const insertOneMacRecordsFromKafkaIntoMako = async (
   await bulkUpdateDataWrapper(oneMacRecordsForMako, "main");
 };
 
+// Seatool sets their days to a fixed time so we just round it to the day.
+function normalizeToDate(timestamp: string | number | Date): number {
+  const date = new Date(timestamp);
```

Comment:
you might want to use UTCDate here

Reply:
I see how it's being used below so that's not entirely necessary, but probably still a good idea because of timezone weirdness